tests/rptest/services/redpanda.py

```diff
@@ -1269,6 +1269,31 @@ def set_cluster_config(self, values: dict, expect_restart: bool = False):
             raise AssertionError(
                 "Nodes report restart required but expect_restart is False")
 
+    def await_feature_active(self, feature_name: str, *, timeout_sec: int):
```

Comment:
What's up with the `*` there?

Reply:
It's the syntax for saying "all args after this must be given as kwargs".  Makes calling python functions less error prone by avoiding positional args.

Reply:
Cool. PEP is [here](https://peps.python.org/pep-0570/#specification) if anyone stumbles upon this thread.